packages/manager/modules/telecom-universe-components/package.json

```diff
@@ -27,6 +27,7 @@
     "angular-translate": "^2.11.0",
     "angular-ui-bootstrap": "~1.3.3",
     "angular-validation-match": "^1.9.0",
+    "bootstrap4": "twbs/bootstrap#v4.0.0",
```

Comment:
Please make sure that hoisted applications are aligned.

```sh
$ yarn why @ovh-ux/ng-ovh-telecom-universe-components
```

Reply:
And you can also update the `BREAKING CHANGE` message to precise that this module now require a new peerDependency.